code/modules/virus2/centrifuge.dm

```diff
@@ -88,21 +99,22 @@
 	return
 
 /obj/machinery/centrifuge/process()
+
 	..()
 
 	if(stat & (NOPOWER|BROKEN))
+		update_icon()
 		return
-	use_power(500)
```

Comment:
If you're removing this, you need to set the machine status to active with the confusingly-named use_power variable which is completely different from the use_power proc. Check machinery.dm for details.

use_power = 1 for idle, use_power = 2 for active